charts/pega/templates/gke/affinity-session-backendconfig.yaml

```diff
@@ -0,0 +1,13 @@
+{{ if (eq .Values.global.provider "gke") }}
+apiVersion: cloud.google.com/v1beta1
+kind: BackendConfig
+metadata:
+  name: gke-affinity-session-backendconfig
+spec:
+  timeoutSec: 40
+  connectionDraining:
+    drainingTimeoutSec: 60
+  sessionAffinity:
+    affinityType: "GENERATED_COOKIE"
+    affinityCookieTtlSec: 600
```

Comment:
This value should not be hardcoded. We should be calculating cookie expiration based on requestor passivation time. We added changes related to this when we added support for alb ingress controller for aws. We can reuse the same value here.

Reply:
Fixed